### PR TITLE
Upgrade consideration for active_series_custom_trackers_config

### PR DESCRIPTION
**Pull request by gubjanos**
#### What this PR does
Adding an upgrade consideration to the replaced config of active_series_custom_trackers_config.
#### Which issue(s) this PR fixes or relates to

Fixes #<issue number>

#### Checklist

- [ ] Tests updated
- [ ] Documentation added
- [ ] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

File: docs/sources/release-notes/v2.1.md [hunk 1]
```diff
@@ -29,6 +29,8 @@ We've updated the default values for 2 parameters in Grafana Mimir to give users
 
 - The store-gateway attributes cache is now enabled by default (achieved by updating the default for `-blocks-storage.bucket-store.chunks-cache.attributes-in-memory-max-items` from `0` to `50000`). This in-memory cache makes it faster to look up object attributes for chunk data. We've been running this optional cache internally for a while and upon a recent configuration audit, realized it made sense to do the same for all users. The increase in store-gateway memory utilization from enabling this cache is negligible and easily justified given the performance gains.
 
+Configuration for `active_series_custom_trackers_config` has been moved from `ingester` section to `limits` section and the former going to be deprecated from Mimir version 2.3 .
```
Comment (pracucci) [inline docs/sources/release-notes/v2.1.md:32]:
```suggestion
- Configuration for `active_series_custom_trackers_config` has been moved from `ingester` section to `limits` section and the former going to be deprecated from Mimir version 2.3 .
```

**Review by pracucci — APPROVED** [review]
LGTM (but let's wait for @09jvilla feedback)

@johannaratliff we'll have to cherry-pick this into `release-2.1`

File: docs/sources/release-notes/v2.1.md (hunk 1 above)
Comment (09jvilla) [inline docs/sources/release-notes/v2.1.md:32]:
```suggestion
As part of the enhancement to make active series custom trackers configurable on a per-tenant basis, we've moved where they get configured. Users of this feature should migrate from defining `active_series_custom_trackers_config` in the `ingester` section of Grafana Mimir's [YAML configuration]({{< relref "../operators-guide/configuring/reference-configuration-parameters/#configuration-parameters" >}}) to defining it in the `limits` section. Both options are supported for now to give users time to make the change, but configuring `active_series_custom_trackers_config` via the `ingester` section has been marked deprecated and will be removed in Mimir 2.3. Note: Configuring `active_series_custom_trackers_config` in both the `limits` and `ingester` sections will cause Grafana Mimir to fail on startup. 
```

**Comment by 09jvilla** [comment]
@pracucci -- @gubjanos is out of office now, so would appreciate if you and @johannaratliff could help me get this one over the line. 

I added a little more information to the original PR to hopefully make it more explicit for users and connect this part of the release notes to the section above where we originally mention the enhancement.

File: docs/sources/release-notes/v2.1.md [hunk 2]
```diff
@@ -29,6 +29,8 @@ We've updated the default values for 2 parameters in Grafana Mimir to give users
...
 
+As part of the enhancement to make active series custom trackers configurable on a per-tenant basis, we've moved where they get configured. Users of this feature should migrate from defining `active_series_custom_trackers_config` in the `ingester` section of Grafana Mimir's [YAML configuration]({{< relref "../operators-guide/configuring/reference-configuration-parameters/#configuration-parameters" >}}) to defining it in the `limits` section. Both options are supported for now to give users time to make the change, but configuring `active_series_custom_trackers_config` via the `ingester` section has been marked deprecated and will be removed in Mimir 2.3. Note: Configuring `active_series_custom_trackers_config` in both the `limits` and `ingester` sections will cause Grafana Mimir to fail on startup.
```
Comment (pracucci) [inline docs/sources/release-notes/v2.1.md:32]: I think needs to be a bullet point, given we're in a list.

```suggestion
- As part of the enhancement to make active series custom trackers configurable on a per-tenant basis, we've moved where they get configured. Users of this feature should migrate from defining `active_series_custom_trackers_config` in the `ingester` section of Grafana Mimir's [YAML configuration]({{< relref "../operators-guide/configuring/reference-configuration-parameters/#configuration-parameters" >}}) to defining it in the `limits` section. Both options are supported for now to give users time to make the change, but configuring `active_series_custom_trackers_config` via the `ingester` section has been marked deprecated and will be removed in Mimir 2.3. Note: Configuring `active_series_custom_trackers_config` in both the `limits` and `ingester` sections will cause Grafana Mimir to fail on startup. 
```

**Review by 09jvilla — APPROVED** [review]
Alrighty - I think we're ready @pracucci !

**Pull request merged**